pkgs/etc-profiles/base.nix

```diff
@@ -35,11 +34,16 @@ in ( derivation {
         packages.nixpkgs-flox.sqlite = {
           meta.outputsToInstall = ["bin" "out" "dev"];
         };
-        packages.nixpkgs-flox.pkg-config = {};
```

Comment:
@aakropotkin - if our "common-paths" module is expecting pkgconfig in the environment then we should add that as a propagatedBuildInput of the `base` derivation.

Reply:
Alex confirms it's just there as a nicety, and shouldn't be added to propagatedBuildInputs by default.